lineage/LineageInputOutput.py

```diff
@@ -203,3 +203,30 @@ def tryRecursion(pColumn, lower, upper, parentCell, currentLineage, lineageSizeI
     # add daughter to current Lineage
     currentLineage.append(daughterCell)
     return daughterCell
+
+def fullRecursive(cell, a):
+    """ To plot the <full> binary tree based on inter-mitotic time (lifetime) of cells.
+    a should be: a = [Clade(lineage1.full_lineage[0].obs[2]+lineage1.full_lineage[0].obs[3])]
+    """
+    if cell.isLeaf():
+        return Clade(branch_length=(cell.obs[2]+cell.obs[3]))
+    else:
+        return Clade(branch_length=(cell.obs[2]+cell.obs[3]), clades=[fullRecursive(cell.left, a), fullRecursive(cell.right, a)])
+
+def CensoredRecursive(cell, a):
+    """ To plot the lineage while censored (from G1 or G2).
+    If cell died in G1, the lifetime of the cell until dies is shown in red.
+    If cell died in G2, the lifetime of the cell until dies is shown in blue.
+    If none of the above, the cell continues to divide and is shown in black.
+    a should be: a = [Clade(lineage1.full_lineage[0].obs[2]+lineage1.full_lineage[0].obs[3])]
+    """
+    if cell.isLeaf():
+        return Clade(branch_length=(cell.obs[2]+cell.obs[3]))
```

Comment:
`cell.time` works for censored ones, too?

Reply:
I have a problem with it. 
![image](https://user-images.githubusercontent.com/34812750/86662372-82018900-bfa1-11ea-95c7-e3a0f42ede09.png)
Have you seen this before? I assume it refers to my edit regarding `cell.time`

Reply:
does cell.time return a floating-point number?

Reply:
Oh, no `cell.time` is the class that stores some other things. Here are the things that it stores:
`cell.time.startT`: the start time of the cell in terms of cumulative running time
`cell.time.endT`: the end time of the cell in terms of cumulative running time
`cell.time.transition_time`: the time between the start time and the G1 transition into G2

It _should_ work for censored ones? But I'm not completely certain about this. It should work for the censored synthetic data. But it might not work on the Heiser data.

Reply:
So `cell.time.endT - cell.time.startT` would give you the total time a cell lived. 

And `cell.time.endT - cell.time.transition_time` would give you the time spent in G2.

And `cell.time.transition_time - cell.time.startT` would give you the time spent in G1.

Reply:
@Farnazmdi So I'm guessing it should be like the following:

```python
         return Clade(branch_length=(cell.time.endT - cell.time.startT)) 
```

Reply:
I just checked, all the values in the `cell.time` subclass are always numbers (never `float('nan')`s). It should work for both censored and uncensored lineages. We might run into some trouble when using real data, but it should work for the synthetic lineages.